.gitignore

```diff
@@ -4,3 +4,5 @@ node_modules/
 
 # Postcss-fied CSS files
 postcss/
+
+.DS_Store
```

Comment:
I'd suggest moving some of the clean-up changes to their own issue so they can be reviewed as one thing.

Reply:
Understood.

Reply:
I've pushed the changes wrt to your comments, Joel. Thanks very much for reviewing.